.github/workflows/performance-audit.yaml

```
@@ -1,29 +1,74 @@
 
+env:
+  ENV: dev
+
 jobs:
   performance-audit:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout code
+      - name: Checkout
         uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)